background_scripts/completion.coffee

```diff
@@ -23,21 +23,28 @@ class Suggestion
     @title ||= ""
     # When @autoSelect is truthy, the suggestion is automatically pre-selected in the vomnibar.
     @autoSelect = false
+    # If @noHighlightTerms is falsy, then we don't highlight matched terms in the title and URL.
+    @noHighlightTerms = false
```

Comment:
Shouldn't this be settable from the constructor, since it is a lifetime property of the suggestion?